terraform/gcp/pipeline/default/main.tf

```diff
@@ -178,11 +184,72 @@ module "postgres_loader_bad" {
   purpose       = "JSON"
   schema_name   = "atomic_bad"
 
-  db_instance_name = module.pipeline_db.connection_name
-  db_port          = module.pipeline_db.port
-  db_name          = var.pipeline_db_name
-  db_username      = var.pipeline_db_username
-  db_password      = var.pipeline_db_password
+  db_instance_name = join("", module.postgres_db.*.connection_name)
+  db_port          = join("", module.postgres_db.*.port)
+  db_name          = var.postgres_db_name
+  db_username      = var.postgres_db_username
+  db_password      = var.postgres_db_password
+
+  # Linking in the custom Iglu Server here
+  custom_iglu_resolvers = local.custom_iglu_resolvers
+
+  telemetry_enabled = var.telemetry_enabled
+  user_provided_id  = var.user_provided_id
+
+  labels = var.labels
+}
+
+# 5. Deploy BigQuery Loader
+module "bad_rows_topic" {
+  source  = "snowplow-devops/pubsub-topic/google"
+  version = "0.1.0"
+
+  count = var.bigquery_db_enabled ? 1 : 0
+
+  name = "${var.prefix}-bq-bad-rows-topic"
+
+  labels = var.labels
+}
+
+resource "google_bigquery_dataset" "bigquery_db" {
+  count = var.bigquery_db_enabled ? 1 : 0
+
+  dataset_id = replace("${var.prefix}_pipeline_db", "-", "_")
```

Comment:
Bit nitpicky, wonder if we should get `sp` or `snowplow` in this name so it's easier to find the snowplow dataset in BQ?

Reply:
Thats fair - ill change pipeline for snowplow!